card/lib/generators/card.rb

```diff
@@ -0,0 +1,43 @@
+# -*- encoding : utf-8 -*-
```

Comment:
generators (and all code) for the decko-rails scenario should go in the decko-rails gem unless there's a solution that involves deeper code sharing.